src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1002,7 +1034,11 @@ private IAIState dumpInventory()
             {
                 getOwnBuilding().alterPickUpPriority(MAX_PRIO);
             }
-            chatSpamFilter.talkWithoutSpam(COM_MINECOLONIES_COREMOD_ENTITY_WORKER_INVENTORYFULLCHEST);
+            if ( worker.getCitizenData() != null )
+            {
+                worker.getCitizenData().triggerInteraction(new StandardInteractionResponseHandler(new TranslationTextComponent(COM_MINECOLONIES_COREMOD_ENTITY_WORKER_INVENTORYFULLCHEST), ChatPriority.IMPORTANT));
```

Comment:
doesnt this constantly add a new interaction?

Reply:
No, first of all its a map, second it has a contains check.

Reply:
kk :D